### PR TITLE
Add run_exports

### PR DESCRIPTION
**Pull request by traversaro**
graphviz is a C  shared library that was also subject to migrations (see https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3457/files), so it make sense to make sure that downstream packages depend correctly on it, to avoid problems such as https://github.com/conda-forge/gazebo-feedstock/issues/159 .

This PR also install libwebp-base on Windows, as without it the Windows build was failing.
 
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by traversaro** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
Looks like this needs webp in the dependencies @traversaro

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,7 +32,7 @@ requirements:
     - libtool                                             # [unix]
     - automake                                            # [unix]
     - autoconf                                            # [unix]
-    - cmake                                               # [win]
+    - cmake 3.25.0                                        # [win]
```
Comment (nehaljwani) [inline recipe/meta.yaml:35]: Hi @traversaro , why is this pin required?

Comment (nehaljwani) [reply recipe/meta.yaml:35]: Could you please add a comment?

Comment (traversaro) [reply recipe/meta.yaml:35]: Sorry, I was debugging the build failure and this was the last attempt before I left the keyboard. Indipendently from the run_exports, it seems that there was a build regression of the recipe on Windows from CMake 3.25.0 to 3.25.1 
 . I will clean up the PR as soon as I am back with a laptop.

**Comment by traversaro** [comment]
Superseded by https://github.com/conda-forge/graphviz-feedstock/pull/117/files .

**Pull request closed without merging**